src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -69,19 +74,20 @@ const PhoneAuthenticator: UISchemaElementComponent<{
 
   const renderExtension = () => (
     showExtension && (
-      <Box
-        width={0.25}
-      >
-        <TextInput
+      <Box width={0.25}>
+        <InputLabel htmlFor="phoneExtension">{getTranslation(translations, 'extension')}</InputLabel>
+        <OutlinedInput
+          value={extension}
           type="text"
-          data-se="extension"
           name="extension"
-          label={getTranslation(translations, 'extension')}
-          value={extension}
-          autocomplete="tel-extension"
-          onChange={(e: ChangeEvent<HTMLInputElement>) => {
+          id="phoneExtension"
+          onChange={(e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
             setExtension(e.currentTarget.value);
           }}
+          inputProps={{
+            'data-se': 'extension',
```

Comment:
Should we try to get this from the transformer layer?

Reply:
Same for the next input in this file

Reply:
Do you mean to like create adhoc FieldElements for Extension & Country? Would those then be attached to `options` of the `(*).phoneNumber` FieldElement?

Reply:
Oh I see this isn't as simple as some of the other places where we can easily spread `data-se` in... Yeah would be trickier. We can leave hardcoded values in here if trying to do the spreading from transformer values adds a lot of complexity.